.travis.yml

```diff
@@ -199,8 +199,9 @@ jobs:
       if: branch = master
       name: Deploy Playground
       env: INSTALL_WORDPRESS=false
+      install:
```

Comment:
How about we override `install` here to skip `npm run build`?

Reply:
`if: branch = master` stops the entire stage from running outside of master, right?

Reply:
Oh I see what you mean, because the playground already runs build. Yeah, good catch!